assets/javascripts/admin_user.js

```diff
@@ -37,4 +37,25 @@ function setup_admin_user() {
             }
         });
     });
+
+    window.deleteUser = function(id){
+        var r = confirm("Are you sure you want to procede deleting this user?");
+        if (r == true) {
```

Comment:
`if (r == true) {`? Seriously?

I would really prefer:
```
if (!confirm(…)) {
    return;
}
…
```

Reply:
Bah, an other copy paste, please this was still in WIP

Reply:
I know but I was wondering why one would write code like this in the first place. And I can't think of any `if (r == true)` checks in our codebase.